source/_integrations/light.mqtt.markdown

```diff
@@ -833,15 +838,15 @@ brightness_template:
   required: false
   type: string
 color_temp_template:
-  description: "[Template](/docs/configuration/templating/#processing-incoming-data) to extract color temperature from the state payload value."
+  description: "[Template](/docs/configuration/templating/#processing-incoming-data) to extract white color temperature from the state payload value."
```

Comment:
Please revert this change, this is not how this term is referred to.

Reply:
I admit I changed it intentionally since I'm not aware about temperature of color (in context of the light). This attribute can be hardly used with other light model but white. Imo it is confusing.

Can you explain why you want to stick with 'color temperature' term?

Reply:
Because it is called color temperature, "white" has nothing to do with that.

Reply:
You are right